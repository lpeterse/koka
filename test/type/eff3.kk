fun test() {
  choose( ref(1), error("hi") )
}

fun test2() {
  choose( const((),ref(1)), ref(2).set(3))
}
