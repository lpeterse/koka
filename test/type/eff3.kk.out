test        : forall<h> () -> <exn,alloc<h>,ndet> ref<h,int>
test2       : () -> ndet ()

